alert-common/src/main/java/com/synopsys/integration/alert/common/util/DateUtils.java

```diff
@@ -24,38 +24,46 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
-import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.util.Date;
-import java.util.TimeZone;
 
 public final class DateUtils {
     public static final String DOCKER_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'";
     public static final String AUDIT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
     public static final String UTC_DATE_FORMAT_TO_MINUTE = "yyyy-MM-dd HH:mm '(UTC)'";
 
-    public static Date createCurrentDateTimestamp() {
-        final ZonedDateTime zonedDateTime = ZonedDateTime.now().withZoneSameInstant(ZoneOffset.UTC);
-        return Date.from(zonedDateTime.toInstant());
+    public static OffsetDateTime createCurrentDateTimestamp() {
+        return OffsetDateTime.now();
     }
 
     public static String createCurrentDateString(String format) {
-        Date currentDate = createCurrentDateTimestamp();
+        OffsetDateTime currentDate = OffsetDateTime.now();
         return formatDate(currentDate, format);
     }
 
-    public static String formatDate(Date date, String format) {
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(format);
-        simpleDateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
-        return simpleDateFormat.format(date);
+    public static String formatDate(OffsetDateTime date, String format) {
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern(format);
```

Comment:
Are we sure this is alright without setting the timezone?

Reply:
We are treating all dates as if they were UTC now, so this should be fine.